src/App.tsx

```diff
@@ -1,61 +1,88 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { UserInfo } from './components/UserInfo';
 
 export const App = () => {
+  const [todos, setTodos] = useState([...todosFromServer]);
+  const users = usersFromServer;
+
+  const [title, setTitle] = useState('');
+  const [selectUser, setSelectUser] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const handlerTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setTitleError(false);
+  };
+
+  const handlerUserChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectUser(event.target.value);
+    setUserError(false);
+  };
+
+  const handSubmit = (event: React.FormEvent) => {
```

Comment:
There is a typo in the function name `handSubmit`. It should be `handleSubmit` for consistency and clarity.